core/domain/suggestion_registry.py

```diff
@@ -377,6 +377,93 @@ def pre_update_validate(self, change):
                 'The new html must not match the old html')
 
 
+class SuggestionTranslateContent(BaseSuggestion):
```

Comment:
Is there a lot of duplication here with other classes? If so, consider perhaps having a nested subclass structure.

Reply:
Please file an issue for that and add a TODO here.

Reply:
Created an issue at #7572.